### PR TITLE
Update searching-issues-and-pull-requests.md

### LINKED ISSUE #9403: "Advanced search syntax" page doesn't include `@me` syntax
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Searching issues and pull requests](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests)

### What part(s) of the article would you like to see updated?

I would like to see a new part of searching manual related to advanced search variables, like `@me`.

### Additional information

1. Open issues search
image

2. Open _Filters_ dropdown and click [**View advanced search syntax**](https://docs.github.com/articles/searching-issues) option
image

3. There is no information on help page about such advanced search option as `commenter:@me`
image

---
### Content design for changes to fix this issue

In the [Searching issues and pull requests](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests) article:

- Create a new section "Searching for my issues and pull requests" after "[Search by a user that's involved in an issue or pull request](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-a-user-thats-involved-in-an-issue-or-pull-request)" as this is the last of a series of sections about filtering on usernames.
- Mention that you can view a list of all the issues and pull requests you created, are assigned, mentioned on, or are requested as a reviewer of (pull requests only) using the **Issues** and **Pull requests** links shown in the top menu bar when you're logged in.
- Explain that you can use `@me` with any of the fields that take a username to limit your search to issues or pull requests that you have interacted with.
- For more information, see "[Understanding the search syntax](https://docs.github.com/en/github/searching-for-information-on-github/getting-started-with-searching-on-github/understanding-the-search-syntax#queries-with-usernames)"
- Give a couple of examples, based on the examples in the sections above.

### PR DESCRIPTION
**Pull request by mark-mxwl**
### Why:

Closes:  [#9403](https://github.com/github/docs/issues/9403). 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added a new section titled, "Search for my issues and pull requests," as proposed and agreed upon in the issue's comments. The section directly addresses the use of `@me`. Examples are provided to maintain consistency within the article.

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

- [X] For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.

- [X] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by steveward** [comment]
Thanks for the PR @mark-mxwl! I'll get this triaged for review. ⚡

File: content/search-github/searching-on-github/searching-issues-and-pull-requests.md [hunk 1]
```diff
@@ -160,6 +160,15 @@ You can use the `involves` qualifier to find issues that in some way involve a c
 | <code>involves:<em>USERNAME</em></code> | **[involves:defunkt involves:jlord](https://github.com/search?q=involves%3Adefunkt+involves%3Ajlord&type=Issues)** matches issues either @defunkt or @jlord are involved in.
 | `in:body` <code>involves:<em>USERNAME</em></code> | [**NOT bootstrap in:body involves:mdo**](https://github.com/search?q=NOT+bootstrap+in%3Abody+involves%3Amdo&type=Issues) matches issues @mdo is involved in that do not contain the word "bootstrap" in the body.
 
+## Search for my issues and pull requests
+
+To quickly view your issues and pull requests, navigate to the respective links from your GitHub dashboard. More advanced search options are available with `@me`, either by using the global search bar, or by filtering results from an individual repository. Any qualifier that accepts a username will allow you to limit your search to issues and pull requests you have created or have interacted with. To do so, follow the desired qualifier with `@me`.
```
Comment (sophietheking) [inline content/search-github/searching-on-github/searching-issues-and-pull-requests.md:165]:
```suggestion
You can search for issues and pull requests you have created or have interacted with by following the desired qualifier with `@me`. Any qualifier that works with a username allows you to limit your search to issues and pull requests you created, are assigned, mentioned on, or are requested as a reviewer of.
```

I think this intro can be a bit shorter. ✨

**Review by sophietheking — DISMISSED** [review]
@mark-mxwl 👋 Thanks for opening this PR and adding this section to the docs, we really appreciate it. I've left one suggestion to shorten the intro.

Please feel free to `@mention` me here to get this merged.

File: content/search-github/searching-on-github/searching-issues-and-pull-requests.md (hunk 1 above)
Comment (mark-mxwl) [reply content/search-github/searching-on-github/searching-issues-and-pull-requests.md:165]: Great suggestion @sophietheking ! This has a much better flow. Thanks so much for the review! 💥

**Review by sophietheking — APPROVED** [review]
🚀

**Pull request merged**